preload.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri May  8 21:09:58 2020
+
+@author: minh
+"""
```

Comment:
What is the purpose of this file?

Reply:
Ma fault, you said it in your commentary. We should not create another file but do it directly in the clean_dataset.py script.